docs/admin-manual/ep-policy-configuration.rst

```diff
@@ -759,16 +688,84 @@ If your Linux distribution uses *systemd*, it will mount the cgroup file
+the amount requested by the job, perhaps rounded up.
+
+This is not something an administrator usually needs to worry about,
+but it does mean that jobs can be monitored with the standard cgroup V2
+tools, like system-cgtop.
+
+By default, HTCondor zeroes out the swap memory limit for the job cgroup,
+because if the job reaches the physical memory limit, and starts paging,
+it will get very slow, and can slow down the system.  The administrator
+can configure HTCondor not to do so, by setting 
+:macro:`DISABLE_SWAP_FOR_JOB` to false.
+
+HTCondor configures cgroupv2 so that any job can further subdivide the memory
+and cpu.  It does so by following the various rules imposed by the kernel.
+The first rule is that Unix processes can only live in the leaf nodes of the
+cgroup tree.  This implies that if a job wants to make a sub-cgroup, it cannot
+simply make a subdirectory of the cgroup it lives in, for then it would live
+in an interior node.  One way to work around this restriction is to create
+a subdirectory, and immediately move the job into that subdirectory. This
+is clunky, and prone to race conditions when the job might be spawning processes
+of its own.  Rather, the starter creates two, nested directories for the job.
+The direct child of the starter's cgroup is the job's "slice".  HTCondor puts
+resource limits on the slice (and implictly on all children of the slice),
+and measures resource usage of the job by measuring the resource of the slice
+(again, and implicitly measuring the sum of the resources of all the child
+subcgroups of that slice).  When it is time to clean up the job, the starter
+removes the slice, and all sub-cgroups thereof.  The starter sets the Unix
+permissions on the slice so that the job can make subdirectories, and thus
+sub-cgroups of the job under the slice, and move processes into those sub-cgroups,
+without violating the kernel's rules about cgroup membership.  However, the
+permissions on the resource limits are set so that the job cannot change them.
+
+The Unix process of the job do not live in the slice, but rather in the
+"scope" cgroup, which is a child of the slice.  This is a leaf node, 
+so the job should never try to make a sub-cgroup of the scope.  This
+naming mimics the naming convention of systemd, but it is HTCondor, not systemd
+which creates and manages this cgroups.
+
+.. mermaid::
+    :caption: Cgroup V2 Organization for jobs
+    :align: center
+
+    flowchart TD
+      starter("`Cgroup 
+                of 
+                condor_starter`")
+      job_scope("`Cgroup
+                  for
+                  job scope`")
+      job_slice("`Cgroup
+                  for
+                  job slice`")
+      starter -->  job_scope
+      job_scope --> job_slice
+      style starter fill:pink
+      style job_scope fill:lightgreen
+      style job_slice fill:lightgreen
+
+In the diagram above, the starter's cgroup is red, as it is not modifiably by 
```

Comment:
The mermaid says "pink", rather than red?

Reply:
"Red" was too dark for the text on top of it.